euclidean_gcd/euclidean_gcd.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import "fmt"
+
+// EuclideanGCDIterative finds GCD of given numbers by iterative method
+func EuclideanGCDIterative(a int, b int) int {
+	for b != 0 {
+		tmp := b
+		b = a % b
+		a = tmp
+	}
+	return a
+}
+
+// EuclideanGCDRecursive finds GCD of given numbers by recursion
+func EuclideanGCDRecursive(a int, b int) int {
+	if b != 0 {
+		return EuclideanGCDRecursive(b, (a % b))
+	}
+	return a
+}
+
+func main() {
+	a := 90
+	b := 65
```

Comment:
Try to give better name to variables. Instead of `a` and `b` , you can use `firstNumber` or `secondNumber` as well.